scripts/update-core-contracts.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# This script updates all flow-core-contracts dependencies to a new version.
+# Specify the desired version as the only argument when running the script:
+#   ./scripts/update-core-contracts.sh v1.2.3
+
+go get github.com/onflow/flow-core-contracts/lib/go/contracts@$1
```

Comment:
very nice! Can we add this to the makefile?

Reply:
Added in https://github.com/onflow/flow-go/pull/3837/commits/7abfc640a697e8d928386d5a8be6becb425402c2.